src/components/popup/popup.component.ts

```diff
@@ -539,29 +539,21 @@ export default class SlPopup extends ShoelaceElement {
   };
 
   render() {
-    return html`
-      <slot name="anchor" @slotchange=${this.handleAnchorChange}></slot>
-
-      <span
+    return html`<slot name="anchor" @slotchange=${this.handleAnchorChange}></slot><span
         part="hover-bridge"
         class=${classMap({
           'popup-hover-bridge': true,
           'popup-hover-bridge--visible': this.hoverBridge && this.active
         })}
-      ></span>
-
-      <div
+      ></span><div
```

Comment:
Prettier re-introduces the line break here and after the "accessible-name" comment in `tooltip.component.ts`, either of which re-introduces the extraneous space.